### PR TITLE
Fix projects

### PR DESCRIPTION
**Pull request by bhush9**
Fixes the code error and test

### CODE REVIEW

**Comment by jriddell** [comment]
makes sense

File: ci-tooling/test/test_projects_factory.rb [hunk 1]
```diff
@@ -41,6 +41,15 @@ def setup
     # fake data in this test which would raise in the adjustment as expections
     # would not be met.
     CI::UpstreamSCM.any_instance.stubs(:releaseme_adjust!).returns(true)
+    stub_request(:get, 'https://projects.kde.org/api/v1/projects/frameworks').
+        with(headers: {'Accept'=>'*/*', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'User-Agent'=>'Ruby'}).
```
Comment (apachelogger) [inline ci-tooling/test/test_projects_factory.rb]: You can drop the entire with part. if you don't define it everything matches, which is what we want here

Comment (bhush9) [reply ci-tooling/test/test_projects_factory.rb]: Fixed.

**Comment by bhush9** [comment]
I'll add that in my todo list to fix at least testcase part.

**Pull request merged**